client/src/components/animal-image-carousel/FilterContainer.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import {
+    FaTimesCircle as IconNothing
+} from "react-icons/fa";
+
+import {
+    GiSharkFin as IconShark,
+    GiCat as IconCat
+} from "react-icons/gi"
+
+const FILTER_TYPES  = {
+    CAT: 'cat',
+    SHARK: 'shark',
+    BOTH: undefined
+}
+
+export default function FilterContainer ({onUpdateFilter, filter}) {
+    return (
+    <div className="filter-container">
+            <IconShark className="filter-icon" onClick={()=>onUpdateFilter('shark')} style={{color: filter === FILTER_TYPES.SHARK? 'orange': 'black'}} />
+            <IconCat className="filter-icon" onClick={()=>onUpdateFilter('cat')} style={{color: filter === FILTER_TYPES.CAT? 'orange': 'black'}}/>
+            <IconNothing className="filter-icon" onClick={()=>onUpdateFilter()} style={{color: filter === FILTER_TYPES.BOTH? 'orange': 'black'}}></IconNothing>
```

Comment:
there seems to be a bug here. can you identify it?

Reply:
I believe this maybe the way I update the filter here, I should always pass one input and maybe instead of undefined I pass it a type 'both' and that way the code is much easier to read.